File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/InlineObject11.scala

package org.openapitools.client.model


case class InlineObject11 (
    /* Group ID to query. */
    _groupId: Long
)
object InlineObject11 {
    def toStringBody(var_groupId: Object) =
        s"""
        | {
        | "groupId":$var_groupId
        | }
        """.stripMargin
}

File: openapi-generator/dart-jaguar/lib/model/inline_object11.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

part 'inline_object11.jser.dart';

class InlineObject11 {
   /* Group ID to query. */
  @Alias('groupId')
  final int groupId;
  

  InlineObject11(
      

{
    
     this.groupId = null 
    }
  );

  @override
  String toString() {
    return 'InlineObject11[groupId=$groupId, ]';
  }
}

@GenSerializer(nullableFields: true)
class InlineObject11Serializer extends Serializer<InlineObject11> with _$InlineObject11Serializer {

}
